### PR TITLE
Homework2

### PR DESCRIPTION
**Pull request by kuzanatoliy**

### CODE REVIEW

File: src/app/shared/search/search.component.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div>
+  <input #searhcInput
+    type="text"
+    [value]="searchValue"
```
Comment (kaseopea) [inline src/app/shared/search/search.component.html]: The task was:
3.	Search control with input and action button.
Place input field, button find and add course button.
Bind [()] input field with component property.
On Find click - console.log input value

Please, use two-way binding as mensioned in the task.

**Review by kaseopea — COMMENTED** [review]
1. Better to name git branches with a sign of current task.
2. Review the Angular styleguide. Especially how to use shared module.
3. Not implemented "Video Course plate" as separate component.

File: src/app/shared/search/search.component.html (hunk 1 above)
Comment (kuzanatoliy) [reply src/app/shared/search/search.component.html]: Changed

File: src/app/courses/course-item/course-item.component.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-course-item',
+  templateUrl: './course-item.component.html',
+  styleUrls: ['./course-item.component.sass']
+})
+export class CourseItemComponent implements OnInit {
+  @Input() course: Object;
+  @Input() index: Number;
```
Comment (kaseopea) [inline src/app/courses/course-item/course-item.component.ts]: No need to set `index` here. Usually you'll be using item id to identify specific item.
For example if you implement pagination - then your indexes can be messed up if you try to delete an item.

Comment (kuzanatoliy) [reply src/app/courses/course-item/course-item.component.ts]: Changed

**Pull request merged**